### ISSUE TITLE
Unable to resolve dependency

### ISSUE THREAD

**Issue by alann-maulana** [sidlatau/flutter_document_picker#11]
### Error
```
Unable to resolve dependency for ':app@dynamicProfile/compileClasspath': Could not resolve project :flutter_document_picker.
Open File
Show Details


Unable to resolve dependency for ':app@dynamicProfileUnitTest/compileClasspath': Could not resolve project :flutter_document_picker.
Open File
Show Details


Unable to resolve dependency for ':app@profile/compileClasspath': Could not resolve project :flutter_document_picker.
Open File
Show Details


Unable to resolve dependency for ':app@profileUnitTest/compileClasspath': Could not resolve project :flutter_document_picker.
Open File
Show Details


Unable to resolve dependency for ':app@dynamicRelease/compileClasspath': Could not resolve project :flutter_document_picker.
Open File
Show Details


Unable to resolve dependency for ':app@dynamicReleaseUnitTest/compileClasspath': Could not resolve project :flutter_document_picker.
Open File
Show Details
```

### Flutter Doctor
```
[✓] Flutter (Channel master, v0.9.7-pre.60, on Mac OS X 10.14 18A391, locale en-ID)
    • Flutter version 0.9.7-pre.60 at /Users/Maulana/Development/flutter
    • Framework revision e438632165 (2 hours ago), 2018-10-05 20:00:11 -0700
    • Engine revision 572fa5646a
    • Dart version 2.1.0-dev.6.0.flutter-c6254163dc

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/Maulana/Development/AndroidSDK
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.0, Build version 10A255
    • ios-deploy 2.0.0
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 29.0.2
    • Dart plugin version 181.5616
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] VS Code (version 1.27.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.18.0

[✓] Connected device (2 available)
    • Android SDK built for x86 • emulator-5554                            • android-x86 • Android 8.1.0 (API 27) (emulator)
    • Eyro Labs iPhone 6        • 20747ee7b210a90b2bf3e80624feb7c635fd59ff • ios         • iOS 12.0

• No issues found!
```

**Comment by sidlatau** [comment]
Looks like it is related to this [#22614](https://github.com/flutter/flutter/issues/22614) Flutter issue.

**Comment by sidlatau** [comment]
I just released plugin version `v1.1.2` with updated dependencies version.

In the app side changing `gradle-wrapper.properties` file last line to `distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip` was needed to compile app successfully.

**Comment by alann-maulana** [comment]
@sidlatau voilaaa...it's working now. Thanks for your kindly help sir. I'm closing this issue for now :)
